react/features/whiteboard/functions.ts

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable import/order */
+import md5 from 'js-md5';
+
+// @ts-ignore
+import { getPinnedParticipant } from '../../features/base/participants';
+import { getRemoteParticipants, isLocalParticipantModerator } from '../base/participants/functions';
+import { appendURLParam } from '../base/util/uri';
+
+// @ts-ignore
+import { getCurrentRoomId, isInBreakoutRoom } from '../breakout-rooms/functions';
+
+import { WHITEBOARD_ID } from './constants';
+
+const getWhiteboardState = (state: any): any => state['features/whiteboard'];
+
+/**
+ * Indicates whether the whiteboard is enabled in the config.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {boolean}
+ */
+export const isWhiteboardEnabled = (state: any): boolean => state['features/base/config'].whiteboard?.enabled;
+
+/**
+ * Indicates whether the whiteboard is open.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {boolean}
+ */
+export const isWhiteboardOpen = (state: any): boolean => getWhiteboardState(state).isOpen;
+
+/**
+ * Indicates whether the whiteboard button is visible.
+ *
+ * @param {Object} state - The state from the Redux store.
+ * @returns {boolean}
+ */
+export const isWhiteboardButtonVisible = (state: any): boolean =>
```

Comment:
Here we need to check if the base URL is set and if the metadata service is available too.

Reply:
Actually, it probably makes more sense to do it in `isWhiteboardEnabled` since without a base URL and a metadata service it cannot work.